shared/cluster.go

```diff
@@ -207,6 +208,25 @@ func FetchClusterIP(namespace, serviceName string) (ip, port string, err error)
 	return ip, port, err
 }
 
+// FetchClusterIPs returns the cluster IPs and port of the service.
+func FetchClusterIPs(namespace, svc string) (ip, port string, err error) {
+	cmd := "kubectl get svc " + svc + " -n " + namespace +
+		" -o jsonpath='{.spec.clusterIPs[*]}' --kubeconfig=" + KubeConfigFile
```

Comment:
also since this can return multiple ips lets fix the signature to return IPS slice
and your caller should be able to parse if the slice sometimes has only one ip
since on your caller u are expecting always more than 1, and there u are doing this slice treatment , for sake of readability and maintainence we should keep signatures clear and concise from whats doing

Reply:
this is not fixed